app/lib/ui/views/account/account_editing_screen.dart

```diff
@@ -129,26 +161,45 @@ class _AccountEditingScreenState extends State<AccountEditingScreen> {
                           ),
                         ),
                         SizedBox(
-                            height: 42,
-                            width: 200,
-                            child: RaisedButton(
-                              onPressed: () {
-                                if (widget.formKey.currentState.validate()) {
-                                  model.saveTransporterAccount(
-                                    userEmailAddress:
-                                        _emailController.text.trim(),
-                                    firstName: _firstNameController.text.trim(),
-                                    lastName: _lastNameController.text.trim(),
-                                    userPhoneNumber:
-                                        _userPhoneNumberController.text.trim(),
-                                  );
-                                }
-                              },
-                              child: Text('Save',
-                                  style: TextStyle(
-                                      fontSize: 20, color: Colors.white)),
-                              color: NavyBlue,
-                            )),
+                          height: 42,
+                          width: 200,
+                          child: RaisedButton(
+                            onPressed: () {
+                              model.selectImage();
+                            },
+                            child: Text('Update Photo',
+                                style: TextStyle(
+                                    fontSize: 20, color: Colors.white)),
+                            color: NavyBlue,
+                          ),
+                        ),
+                        Padding(
+                          padding: EdgeInsets.only(
+                            top: 20.0,
+                          ),
+                        ),
+                        SizedBox(
+                          height: 42,
+                          width: 200,
+                          child: RaisedButton(
+                            onPressed: () {
+                              if (widget.formKey.currentState.validate()) {
+                                model.saveTransporterAccount(
```

Comment:
- [ ] Missing `displayImageUrl` when saving here

Reply:
The image url is not being added in the UI screen, but instead in viewModel. The path of the selected image would be in the business logic. Please refer to the tutorial I followed

Reply:
Hm. OK. It's code-smelly, but yes that's how it works.